docs/content/guides/dagster/airbyte-managed-elements.mdx

```diff
@@ -0,0 +1,159 @@
+
+airbyte_instance = airbyte_resource.configured(
+    {
+        "host": "localhost",
+        "port": "8000",
+        # If using basic auth, include username and password:
+        "username": "airbyte",
+        "password": {"env": "AIRBYTE_PASSWORD"},
+    }
+)
+
+airbyte_reconciler = AirbyteManagedElementReconciler(
+    airbyte=airbyte_instance,
+    connections=[],
+)
+```
+
+## 2. Defining sources and destinations
+
+Next, we will define our sources and destinations. Sources and destinations can be constructed manually using the `AirbyteSource` and `AirbyteDestination` classes, respectively, however `dagster-airbyte` also provides generated, typed classes for specific source and destination types. See the [API docs](/\_apidocs/libraries/dagster-airbyte#managed-config-generated-sources) for more information on these generated classes. In this case, we'll import the generated classes for the `File` source and `Local JSON` destination.
+
+Here, we'll configure a source that reads from a hosted CSV file and a destination which writes it to a local JSON file.
+
+```python startafter=start_define_sources endbefore=end_define_sources file=/guides/dagster/managed_elements/airbyte.py dedent=4
+from dagster_airbyte.managed.generated.sources import FileSource
+from dagster_airbyte.managed.generated.destinations import LocalJsonDestination
+
+cereals_csv_source = FileSource(
+    name="cereals-csv",
+    url="https://docs.dagster.io/assets/cereal.csv",
+    format="csv",
+    provider=FileSource.HTTPSPublicWeb(),
+    dataset_name="cereals",
+)
+
+local_json_destination = LocalJsonDestination(
+    name="local-json",
+    destination_path="/local/cereals_out.json",
+)
+```
+
+## 3. Defining a connection
+
+Finally, we will define a connection between our source and destination. This is defined using the `AirbyteConnection` class.
+
+```python startafter=start_define_connection endbefore=end_define_connection file=/guides/dagster/managed_elements/airbyte.py dedent=4
+from dagster_airbyte import AirbyteConnection, AirbyteSyncMode
+
+cereals_connection = AirbyteConnection(
+    name="download-cereals",
+    source=cereals_csv_source,
+    destination=local_json_destination,
+    stream_config={"cereals": AirbyteSyncMode.FULL_REFRESH_OVERWRITE},
+)
+```
+
+We'll supply our new connection to the reconciler we defined earlier:
+
+```python startafter=start_new_reconciler endbefore=end_new_reconciler file=/guides/dagster/managed_elements/airbyte.py dedent=4
+airbyte_reconciler = AirbyteManagedElementReconciler(
+    airbyte=airbyte_instance,
+    connections=[cereals_connection],
+)
+```
+
+## 4. Validating changes
+
+Next, we'll use the `dagster-me` CLI to sanity check our reconciler and apply any changes. The `check` command will print out any differences between the current state of the Airbyte instance and the desired state specified in the reconciler. The CLI can be invoked by pointing it at a module which contains the reconciler.
```

Comment:
```suggestion
Next, we'll use the `dagster-me` CLI to sanity-check our reconciler and apply any changes. The `check` command prints out differences between the current state of the Airbyte instance and the desired state specified in the reconciler. The CLI can be invoked by pointing it at a module that contains the reconciler.
```

A few small grammar tweaks here. Also suggesting active voice to improve clarity / readability.